src/components/collapsible-section/collapsible-section.scss

```diff
@@ -125,38 +124,46 @@ header {
     flex-shrink: 0;
 }
 
-@keyframes fade-in-body {
-    0% {
-        opacity: 0;
-    }
-    100% {
-        opacity: 1;
-    }
-}
-
 .body {
-    animation: fade-in-body 0.3s ease-in forwards;
     background-color: var(--body-background-color, var(--contrast-100));
-    padding-left: var(--body-padding, functions.pxToRem(20));
-    padding-right: var(--body-padding, functions.pxToRem(20));
-    margin-bottom: functions.pxToRem(24);
```

Comment:
This was a problematic and wrong choice from the beginning. Now removed, since it was messing up the UI after the change. This is not a breaking change, and nothing that anyone will miss or will notice. If they miss it for some weird reason, they can always add it back by writing where the component is consumed.

```scss
limel-collapsible-section[is-open] {
	margin-bottom: 1.5rem;
}
```